tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -762,6 +769,29 @@ EOF
 	info "Create /etc/resolv.conf file in rootfs if not exist"
 	touch "$dns_file"
 
+    if [ -n "${TEE_PLATFORM}" ]; then
+		if [ "${TEE_PLATFORM}" == "es" ]; then
+			info "Adding agent config for ${TEE_PLATFORM}"
+			#KBC URI will be specified in the config file via kernel params
+			AA_KBC_PARAMS="online_sev_kbc::123.123.123.123:44444" envsubst < "${script_dir}/agent-config.toml.in" | tee "${ROOTFS_DIR}/etc/agent-config.toml"
```

Comment:
I'm not sure this matters anymore. Now the annotation will override the default value in the agent config file. I think we're removing the agent config file anyway so maybe we shouldn't bother with this.

Reply:
removed, thanks.